app/controllers/api/v1/bulk_download_controller.rb

```diff
@@ -187,19 +242,33 @@ def summary
       def generate_curl_config
         valid_accessions = []
         file_ids = []
-        # sanitize study accessions and file types
-        if params[:file_ids]
+        tdr_files = {}
+
+        # branch based on whether they provided a download_id, file_ids, or accessions
+        if params[:download_id]
+          download_req = DownloadRequest.find(params[:download_id])
+          if !download_req
+            render json: {error: 'Invalid download_id provided'}, status: 400 and return
```

Comment:
@eweitz that isn't technically true - Calling `Model.find(id)` where `id` is either `nil` or a non-existent ID will both return `nil`.  This is standard for both `ActiveRecord`, and we have configured `Mongoid` to return `nil` rather than throw an error via [this line](https://github.com/broadinstitute/single_cell_portal_core/blob/development/config/mongoid.yml#L118) in our `Mongoid` configuration.  As such I think error message is actually appropriate.

Reply:
Ah, right.  I think I somehow conflated `download_req` and `download_id`, and thus misunderstood.